cmd/lakefs/cmd/run.go

```diff
@@ -89,7 +90,7 @@ func NewAuthService(ctx context.Context, cfg *config.Config, logger logging.Logg
 		}
 		return auth.NewMonitoredAuthServiceAndInviter(apiService)
 	}
-	authService := auth.NewAuthService(
+	authService := acl.NewAuthService(
```

Comment:
Not sure about it, but it feels kinda weird that `acl` is creating the `auth` service:
I think that `acl` is part of `auth` (is it?), so maybe rename the package to just `contrib/auth`?

Reply:
This is an ACL auth service implementation - It's not going to stay this way. 
The next step will be to take it out completely and run it as an external server so this code is temporary